sys/hacl/build.rs

```diff
@@ -25,34 +30,42 @@ fn includes(home_dir: &Path) -> Vec<String> {
 }
 
 fn append_simd128_flags(flags: &mut Vec<String>) {
-    // Platform detection
-    if cfg!(simd128) {
-        flags.push("-DSIMD128".to_string());
-        if cfg!(any(target_arch = "x86", target_arch = "x86_64")) {
-            flags.push("-msse4.1".to_string());
-            flags.push("-msse4.2".to_string());
-            flags.push("-mavx".to_string());
-        }
+    if cfg!(any(target_arch = "x86", target_arch = "x86_64")) {
+        flags.push("-msse4.1".to_string());
```

Comment:
These flags probably won't work on Windows. We need to make this msvc compatible.

Reply:
msvc uses to ignore such flags, I used a proper guard to push these flags where necessary.

Reply:
MSVC builds all this without any flags? It looks like it's building but I'm a little surprised. But let's get it in like this. We'll see if it works as expected once we use it.

Reply:
Yes, I made sure that simd128/256 code are built on MSVC.